Formplayer/testPages/basic_test_app/basic_test_app_preview.py

```diff
@@ -1635,6 +1680,93 @@ def verify_pagination(self):
         self.driver.back()
         self.switch_to_default_content()
 
+    def turn_off_one_question_per_screen(self):
+        self.switch_to_frame(self.iframe)
+        self.wait_to_click(self.settings_option)
+        time.sleep(2)
+        isChecked = self.is_present(self.turn_on_one_question_toggle_button)
+        print(isChecked)
+        time.sleep(2)
+        if isChecked is True:
+            # self.driver.execute_script("arguments[0].click();", self.driver.find_element(By.XPATH, "//th[(text()='Use one question per screen')]/following-sibling::td//input"))
+            self.wait_to_click(self.toggle_button_one_question)
+            time.sleep(10)
+            print("Toggled OFF")
+            self.wait_to_click(self.done_button)
+        self.switch_to_default_content()
+
+    def maps_record_location(self):
+        self.switch_to_frame(self.iframe)
+        time.sleep(5)
+        self.send_keys(self.location_input, UserData.map_input)
+        time.sleep(5)
+        self.wait_to_click(self.location_search_button)
+        time.sleep(5)
+        self.wait_to_click(self.clear_button)
+        time.sleep(3)
+        assert self.is_present(self.blank_latitude), "Coordinates not cleared"
+        time.sleep(3)
+        self.wait_to_click(self.submit_form_button_2)
+        time.sleep(10)
+        self.switch_to_default_content()
+
+    def sub_menus(self):
+        self.switch_to_frame(self.iframe)
+        time.sleep(5)
+        self.wait_to_click(self.start_option)
+        self.wait_to_click(self.parent_menu)
+        self.is_present_and_displayed(self.parent_survey)
+        self.is_present_and_displayed(self.child_menu)
+        self.is_present_and_displayed(self.visible_child_survey)
```

Comment:
instead of 
`self.switch_to_frame(self.iframe)
        time.sleep(5)
        self.wait_to_click(self.start_option)
        self.wait_to_click(self.parent_menu)
        self.is_present_and_displayed(self.parent_survey)
        self.is_present_and_displayed(self.child_menu)
        self.is_present_and_displayed(self.visible_child_survey)`

you could have used:
`       self.open_form(self.parent_menu)
        self.is_present_and_displayed((By.XPATH, self.case_list_menu.format(self.visible_child_survey)))
        self.is_present_and_displayed((By.XPATH, self.case_list_menu.format(self.child_menu)))`